### PR TITLE
processors: add targz processor

### PR DESCRIPTION
**Pull request by setrofim**
Add an output processor to create a tarball of WA output directory.

### CODE REVIEW

File: wa/processors/targz.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+class TargzProcessor(ResultProcessor):
+
+    name = 'targz'
+
+    description = '''
+    Create a tarball of the output directory.
+
+    This will create a gzip-compressed tarball of the outputdirectory. By
```
Comment (marcbonnici) [inline wa/processors/targz.py]: Missing space "outputdirectory"

File: wa/processors/targz.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    default, it will be created at the same level and will have the same name
+    as the output directory but with a .tar.gz extensions.
+    '''
+
+    parameters = [
+        Parameter('outfile',
+                  description='''
+                  The name  of the output file to be used. If this is not an
+                  absolute path, the file will be created realtive to the
+                  directlory in which WA was invoked. If this contains
```
Comment (marcbonnici) [inline wa/processors/targz.py]: "directlory" -> "directory"

File: wa/processors/targz.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                  subdirectories, they must already axist.
```
Comment (marcbonnici) [inline wa/processors/targz.py]: "axist" -> "exist"

File: wa/processors/targz.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+                  The name may contain named fromat specifiers. Any of the
```
Comment (marcbonnici) [inline wa/processors/targz.py]: "fromat" -> "format"

**Comment by setrofim** [comment]
Comments addressed.

**Pull request merged**